setup.py

```diff
@@ -221,7 +221,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'sphinxcontrib-httpdomain>=1.7.0',
     "sphinxcontrib-redoc>=1.6.0",
     "sphinxcontrib-spelling==5.2.1",
-    f"sphinx-airflow-theme @ {_SPHINX_AIRFLOW_THEME_URL}",
+    # f"sphinx-airflow-theme @ {_SPHINX_AIRFLOW_THEME_URL}",
```

Comment:
```suggestion
    f"sphinx-airflow-theme @ {_SPHINX_AIRFLOW_THEME_URL}",
```

Guessing this was debugging?

Reply:
Yes, mb.